### DISCUSSION TITLE
Login problems

### DISCUSSION THREAD

**Discussion by Froc88** [rejetto/hfs#441] [General]
Hi!
This error appears when logging in
"login_bad_cookies"
I use nginx with ssl a as a reverse proxy.
I've had the site for a long time and I haven't made any changes.
![image](https://github.com/rejetto/hfs/assets/129078816/d72746f2-d006-4db2-a59c-719cc38106ec)
The nginx log shows
"POST /~/api/loginSrp2 HTTP/1.1" 409 31
If I access the IP locally, it works perfectly

Any idea?
Thanks!

**Comment by rejetto** [comment]
what version?

**Comment by Froc88** [comment]
0.50.5
0.50.6

**Comment by rejetto** [comment]
are you using nginx on https but hfs on http?

**Reply by Froc88** [reply]
I have a subdomain with letsencrypt. Using nginx as reverse proxy

**Reply by rejetto** [reply]
ok but that doesn't answer my question

**Reply by Froc88** [reply]
Hfs on http

**Reply by rejetto** [reply]
And https on ngnix?

**Reply by Froc88** [reply]
Yes.

**Comment by rejetto** [comment]
is it possible that you accessed hfs through nginx both with https and http ?

**Comment by rejetto** [comment]
try adding this on your nginx `X-Forwarded-Proto: https`

**Comment by Froc88** [comment]
![image](https://github.com/rejetto/hfs/assets/129078816/72b9b492-ab8c-4fc7-9fda-b094e8e84778)
I left those settings.
Still the same error.
I have tried creating a subdomain without https with these configurations and it gives the same error
![image](https://github.com/rejetto/hfs/assets/129078816/9aace73a-5b94-478c-a228-545fd4ceb03e)

**Comment by rejetto** [comment]
Do you have the same problem if you bypass nginx and use HFS directly?

**Reply by Froc88** [reply]
If I access the IP locally it works fine

**Comment by rejetto** [comment]
can i try to login to your server and see the problem myself?

**Reply by Froc88** [reply]
Yes, no problem
Where do I send the url and the user?

**Reply by rejetto** [reply]
a@rejetto.com

**Reply by Froc88** [reply]
I was able to solve it, it was a plugin problem.
I deleted them all, then it worked properly.
Thanks for your attention!

**Comment by rejetto** [comment]
ok, it's a pity we don't know which one was it
